.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -36,6 +36,9 @@ check the checkboxes as you progress through this list.
 - [ ] Gave the onboarded developer push access to the Azul Repository on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
 - [ ] Gave the onboarded developer access to development CI/CD systems on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
 - [ ] Gave the onboarded developer [GCP Credentials](https://console.cloud.google.com/)
+- [ ] Added the onboarded developer's Google workspace account to the `azul-dev` group in Terra
```

Comment:
Please add an entry for obtaining the certifications required to handle managed-access data. Ideally, there would be a link to the certifications, or a link to a document listing them.